### ISSUE TITLE
Sample bank names are a bit unweildy

### ISSUE THREAD

**Issue by yaxu** [tidalcycles/strudel#862]
Bank names are quite a lot to remember and type
```javascript
sound("bd hh sd oh").bank("RolandTR909")
```

Shorter names could be better, maybe as aliases:
```javascript
sound("bd hh sd oh").bank("tr909")
```

Being able to put them in the sound list could be nice too, replacing the current `_`:
```javascript
sound("tr909:bd tr606:sd")
```

**Comment by felixroos** [comment]
yep it's annoying to write those long drum machine names all the time

> Being able to put them in the sound list could be nice too, replacing the current _:

the list notation with `sound` is already used for `s:n:gain`

**Comment by felixroos** [comment]
somewhat related: https://github.com/tidalcycles/strudel/issues/592

**Comment by daslyfe** [comment]
I think it could be cool to have a dedicated function that lets you pattern through all of the banks of drum machines. so you could do like `sound("bd hh sd oh").drummachine("1")`

**Comment by lwlsn** [comment]
Maybe adding an autocomplete feature for drum bank samples??
